lib/ldap_fluff/ad_member_service.rb

```diff
@@ -20,7 +20,7 @@ def _groups_from_ldap_data(payload)
     data = []
     if !payload.nil?
       first_level  = payload[:memberof]
-      total_groups = _walk_group_ancestry(first_level)
+      total_groups, b = _walk_group_ancestry(first_level, first_level)
```

Comment:
a) Replace `b` with `_` if you're discarding the variable.
b) The equals signs aren't lined up any more.
c) The extra argument to `_walk_group_ancestry` isn't declared in its method.